CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 16.3.0 (09/11/2024)
+
+### Out-of-band user creation
+
+Cluster administrators are now able to configure Teleport's `ssh_service` to
+ensure that certain host users exist on the machine without the need to start
+an SSH session.
+
+### Other improvements and fixes
+
+* Allow the cluster wide ssh dial timeout to be set via `auth_service.ssh_dial_timeout` in the Teleport config file. [#46507](https://github.com/gravitational/teleport/pull/46507)
+* Fixed an issue preventing session joining while host user creation was in use. [#46501](https://github.com/gravitational/teleport/pull/46501)
+* Added host users that are created out-of-band, independently of an SSH session. [#46498](https://github.com/gravitational/teleport/pull/46498)
```

Comment:
I think this is basically the same as "out of band user creation" already mentioned in the separate section. Might just insert the PR link there.

Reply:
Done